parsons/google/google_bigquery.py

```diff
@@ -814,16 +790,14 @@ def copy(
         job_config.schema = schema
 
         gcs_client = gcs_client or GoogleCloudStorage()
-        temp_blob_name = f"{uuid.uuid4()}.csv"
+        temp_blob_name = f"{uuid.uuid4()}.{data_type}"
         temp_blob_uri = gcs_client.upload_table(tbl, tmp_gcs_bucket, temp_blob_name)
 
         # load CSV from Cloud Storage into BigQuery
         try:
-            self.copy_from_gcs(
-                gcs_blob_uri=temp_blob_uri,
-                table_name=table_name,
-                if_exists=if_exists,
-                max_errors=max_errors,
+            self._load_table_from_uri(
+                source_uris=temp_blob_uri,
+                destination=self.get_table_ref(table_name=table_name),
                 job_config=job_config,
                 **load_kwargs,
             )
```

Comment:
@Jason94 I think this might be the source of the bug you reported in slack. `copy` used to call the `copy_from_gcs` under the hood which meant it used the defaults set in the `copy_from_gcs` function signature, which are different from the defaults of the underlying google api

I think our options are:
1. add those same parameters and defaults to the `copy` function
2. modify the `copy` function to still call `copy_from_gcs` under the hood

I think I'd vote number 1. By calling the lower level `_load_table_from_uri` function, it made it a little easier to manage the schema stuff but we could probably figure it out.

Reply:
Yeah, I agree. #1 seems fine to me.

Reply:
For reference, here's the script:

```python
import dotenv
from parsons import Table
from parsons.google.google_bigquery import GoogleBigQuery

dotenv.load_dotenv(override=True)


def main():
    bq = GoogleBigQuery()

    data = Table([{"id": 1, "text": "Test\ntext"}])

    bq.copy(data, "test.newlines", if_exists="drop")


if __name__ == "__main__":
    main()
```